instrumentation/http-url-connection/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/httpurlconnection/HttpMethodAttributeExtractor.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.httpurlconnection;
+
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import java.net.HttpURLConnection;
+import javax.annotation.Nullable;
+
+public class HttpMethodAttributeExtractor<
+        REQUEST extends HttpURLConnection, RESPONSE extends Integer>
+    implements AttributesExtractor<REQUEST, RESPONSE> {
+
+  private HttpMethodAttributeExtractor() {}
+
+  public static AttributesExtractor<? super HttpURLConnection, ? super Integer> create() {
+    return new HttpMethodAttributeExtractor<>();
+  }
+
+  @Override
+  public void onStart(AttributesBuilder attributes, Context parentContext, HttpURLConnection o) {}
```

Comment:
we (I?) love long names 😄
```suggestion
  public void onStart(AttributesBuilder attributes, Context parentContext, HttpURLConnection connection) {}
```